README.md

```diff
@@ -1,6 +1,7 @@
 # TEAM ID
-This is a team repo for members of the team {Team ID}
+This is a team repo for members of the team {Team T17-A2}
 
 # Team Members
 * [Jane Doe](members/janeDoe.md)
 * [John Doe](members/johnDoe.md)
```

Comment:
Hi the LO tell me write comments so im here

Reply:
Eh whaddup